tiny_dnn/core/framework/tensor.h

```diff
@@ -351,16 +277,20 @@ class Tensor {
    * matrix view from
    *                                                     // offset 4.
    */
-  Tensor subView(std::initializer_list<size_t> const &start,
-                 std::initializer_list<size_t> const &new_shape) {
-    return subview_impl(start, new_shape);
+  template <class... Ranges>
+  Tensor subView(Ranges... ranges) {
+    return Tensor(xt::view(storage_, from_init_lists(ranges)...));
```

Comment:
no it doesn't since init list can't be forwarded (at least I think that's the reason)